package.json

```diff
@@ -30,15 +30,14 @@
     "test": "npm run lint && npm run spec"
   },
   "engines": {
-    "node": ">=6.0.0"
+    "node": ">= 8.0.0"
   },
   "dependencies": {
     "debug": "^4.1.1",
-    "electron-installer-common": "^0.6.2",
+    "electron-installer-common": "electron-userland/electron-installer-common#drop-node-6",
     "fs-extra": "^7.0.1",
```

Comment:
This also needs updating, v8 dropped support for node 6

Reply:
> v8 dropped support for node 6

[It didn't](https://github.com/jprichardson/node-fs-extra/blob/8.0.0/CHANGELOG.md#800--2019-05-11). I've generally been hesitant to upgrade because I've been hitting bugs with fs-extra 8.0.0 and 8.0.1 such as https://github.com/jprichardson/node-fs-extra/issues/684.